L/LLVM/common.jl

```diff
@@ -160,7 +160,7 @@ if [ -z "${LLVM_WANT_STATIC}" ]; then
     CMAKE_FLAGS+=(-DLLVM_SHLIB_SYMBOL_VERSION:STRING="JL_LLVM_${LLVM_MAJ_VER}.${LLVM_MIN_VER}")
 fi
 
-if [[ "${target}" == *linux* || "${target}" == *mingw* ]]; then
+if [[ "${target}" == *linux* || "${target}" == *mingw* && "${bb_full_target}" != *sanitize* ]]; then
```

Comment:
I guess I should have put the condition `"${bb_full_target}" != *sanitize*` earlier, right?